packages/components/src/font-size-picker/index.js

```diff
@@ -131,12 +132,16 @@ function FontSizePicker(
 				className={ `${ baseClassName }__header` }
 			>
 				<FlexItem>
-					{ __( 'Size' ) }
-					{ headerHint && (
-						<span className={ `${ baseClassName }__header__hint` }>
-							{ headerHint }
-						</span>
-					) }
+					<BaseControl.VisualLabel>
```

Comment:
Curious about if we could use polymorphism and do something like `FlexItem as={ BaseControl.VisualLabel }` ?

Reply:
Good question. I wonder if it's just me, but I'm uncomfortable using a custom component in an `as` prop. It's hard to anticipate what the result is going to be, with regard to conflicting props/styles. At least, you can't get the full picture without looking at all the props/styles on both components, which will continue to be a tax and a risk for anyone working with those parts down the line. So I think I would avoid it unless necessary 🤔 Do you see benefits, whether in this specific case, or in the general case?

Reply:
I guess the main reasons why I asked are:

- from a more practical point of view, to keep the DOM structure leaner
- from a "idealistic" perspective, composing two component that way feels elegant

But I totally understand and _share_ your points — I'm definitely ok keeping the code as-is.